superset/assets/package.json

```diff
@@ -206,6 +206,7 @@
     "typescript": "^3.1.3",
     "url-loader": "^1.0.1",
     "webpack": "^4.19.0",
+    "webpack-merge": "^4.2.1",
```

Comment:
this looks great!

this dependency is to enable the merging in the case that a custom `webpack.config.js` exists yeah? since this dep wouldn't actually appear in `incubator-superset` when you search, I'd just want to make sure this isn't removed unintentionally by someone in the future (which would then break our build). To prevent that I'm wondering if we should either:

- move the dependency entirely to our build config (running e.g., `npm install && npm install webpack-merge && npm run build --config=...`), or
- add a test to ensure that the overriding works, or
- add an example custom `webpack.config.js` that's always used, that could just export the default one 🤔 (this would also provide an example for others who want to use this functionality)

Reply:
Yes webpack-merge is added for our custom webpack.config.js. Moving it into our build config seems like a good idea. 👍